ticket_first_and_last_touch.view.lkml

```diff
@@ -0,0 +1,206 @@
+view: ticket_first_and_last_touch {
+  derived_table: {
+    sql: SELECT
+        distinct ticket_id,
+        FIRST_VALUE(CASE
+                      WHEN audits.metadata__system__location LIKE '%Philippines%' THEN 'Philippines'
+                      WHEN audits.metadata__system__location LIKE '%San Francisco%' THEN 'San Francisco'
+                      WHEN audits.metadata__system__location LIKE '%Mountain View%' THEN 'San Francisco'
+                      WHEN audits.metadata__system__location LIKE '%Englewood, CO%' THEN 'Mexico'
+                      WHEN audits.metadata__system__location LIKE '%Mexico%' THEN 'Mexico'
+                      WHEN audits.metadata__system__location LIKE '%United States%' THEN 'United States'
+                      WHEN audits.metadata__system__location LIKE '%Canada%' THEN 'Canada'
+                      WHEN audits.metadata__system__location LIKE '%United Kingdom%' THEN 'UK'
+                      WHEN audits.metadata__system__location LIKE '%United Arab Emirates%' THEN 'UAE'
+                      ELSE 'Other' END) OVER (PARTITION BY ticket_id ORDER BY audits.created_at ASC) AS first_touch_agent_location,
+        FIRST_VALUE(users.name) OVER (PARTITION BY ticket_id ORDER BY audits.created_at ASC) AS first_touch_agent_name,
```

Comment:
[BLOCKING] Can we use IDs rather than hardcoded names?

Reply:
Yes, definitely.  Updated.